### PR TITLE
Some random fixes and hacks for OOS 10

### PR DESCRIPTION
**Pull request by amog787**
Check each commit for details.

### CODE REVIEW

**Comment by MacTavishAO** [comment]
@erfanoabdi looks fine to me.

File: roms/10/OxygenOS/system/bin/dt2w.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/system/bin/sh
+
+# Scrape in background for change in switch state
+logcat -c && logcat -b events -e "double_click_light_screen_key\,[01]" | while read line
```
Comment (amog787) [inline roms/10/OxygenOS/system/bin/dt2w.sh:4]: Doesn't affect deep sleep or battery in any way, tested on K20, K20P and X2

Comment (erfanoabdi) [reply roms/10/OxygenOS/system/bin/dt2w.sh:4]: this is dirty lol

**Comment by erfanoabdi** [comment]
i'm fine with all except dt2w

File: roms/10/OxygenOS/system/bin/dt2w.sh (hunk 1 above)
Comment (amog787) [reply roms/10/OxygenOS/system/bin/dt2w.sh:4]: Welp ¯\_(ツ)_/¯

**Comment by amog787** [comment]
> i'm fine with all except dt2w

Removed dt2w

**Comment by erfanoabdi** [comment]
thanks 👍

**Pull request merged**